app/models/upload_file.rb

```diff
@@ -0,0 +1,72 @@
+# app/models/upload_file.rb
+class UploadFile < Item
+    def edit(_count)
+      {
+        action: 'edit',
+        elements: [
+          {
+            type: 'link',
+            text: 'Remove',
+            href: "/questions/#{id}",
+            method: 'delete'
+          },
+          {
+            type: 'input',
+            input_type: 'text',
+            name: "item[#{id}][seq]",
+            id: "question_#{id}_seq",
+            value: seq.to_s
+          },
+          {
+            type: 'input',
+            input_type: 'text',
+            name: "item[#{id}][id]",
+            id: "question_#{id}",
+            value: id.to_s
+          },
+          {
+            type: 'textarea',
+            cols: 50,
+            rows: 1,
+            name: "item[#{id}][txt]",
+            id: "question_#{id}_txt",
+            placeholder: 'Edit item content here',
+            value: txt
+          },
+          {
+            type: 'input',
+            input_type: 'text',
+            size: 10,
+            name: "item[#{id}][question_type]",
+            id: "question_#{id}_question_type",
+            value: question_type,
+            disabled: true
+          }
+        ]
+      }.to_json
+    end
+  
+    def view_question_text
```

Comment:
view_item_text  … make this change globally; I won't note future occurrences of this issue.